### PR TITLE
fixed some punctuations and articles

### PR DESCRIPTION
**Pull request by pssukhwani**
Changes in this pull request:

Fixes:
- Articles error: Online number: 224 - after the word  "choose" there should be a "the".
- Punctuations error:
  - Online number: 231 - in the word "webapp", there should be a space like "web app".
  - Online number: 231- after the word "tutorial" there should be a comma ",".
  - Online number: 266- after the word " message on there" there should be a comma ",".

### CODE REVIEW

File: en/deploy/README.md [hunk 1]
```diff
@@ -221,14 +221,14 @@ Now our code is on PythonAnywhere, our virtualenv is ready, and the database is
 
 Click back to the PythonAnywhere dashboard by clicking on its logo, and then click on the **Web** tab. Finally, hit **Add a new web app**.
 
-After confirming your domain name, choose **manual configuration** (N.B. – *not* the "Django" option) in the dialog. Next choose **Python 3.6**, and click Next to finish the wizard.
+After confirming your domain name, choose the **manual configuration** (N.B. – *not* the "Django" option) in the dialog. Next, choose **Python 3.6**, and click Next to finish the wizard.
```
Comment (jhodgdon-drp) [inline en/deploy/README.md]: The additional "the" here, to me, seems very awkward. The comma is probably a good idea.

Comment (jhodgdon-drp) [reply en/deploy/README.md]: Actually, this sentence doesn't seem to be in the file any more?

File: en/deploy/README.md [hunk 2]
```diff
@@ -221,14 +221,14 @@ Now our code is on PythonAnywhere, our virtualenv is ready, and the database is
...
 
 > **Note** Make sure you choose the "Manual configuration" option, not the "Django" one. We're too cool for the default PythonAnywhere Django setup. ;-)
 
 
 ### Setting the virtualenv
 
-You'll be taken to the PythonAnywhere config screen for your webapp, which is where you'll need to go whenever you want to make changes to the app on the server.
+You'll be taken to the PythonAnywhere config screen for your web app, which is where you'll need to go whenever you want to make changes to the app on the server.
```
Comment (jhodgdon-drp) [inline en/deploy/README.md]: This sentence isn't in the file any more either.

**Comment by jhodgdon-drp** [comment]
It looks like all the changes in this pull request are obsolete -- none of the lines changed seem to be present in the deploy/README.md file at this time. So, this PR should probably just be closed?

**Comment by ekohl** [comment]
Thanks for your PR. I'm going to close this because it's outdated. We'd welcome any further PRs with improvements.

**Pull request closed without merging**